### DISCUSSION TITLE
Changing the value of the electrolyte conductivity

### DISCUSSION THREAD

**Discussion by MatHunt** [pybamm-team/PyBaMM#2530] [Q&A]
I understand that electrolyte conductivity is a function. I would like to scale this to see how it affects the results. I can't seem to simply multiply it by a number as it appears to be a function. I'm aware that I can define a new function and use that, but I simply want to see how to change the function via a scaling.

**Comment by valentinsulzer** [comment]
Define a function like this
```python3
def conductivity(c, T):
    return pybamm.Scalar("scale") * (...)
```
then you can change `scale` to different numbers

**Reply by MatHunt** [reply]
I'm not sure this works thinking about it. I don't think you can simply swap functions in and out without knowing the argument first. I think that you have to explicitly define a function and then scale the coefficients.
